apm-sniffer/apm-sdk-plugin/httpClient-4.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/httpClient/v4/HttpClientExecuteInterceptor.java

```diff
@@ -112,9 +112,14 @@ public void handleMethodException(EnhancedInstance objInst, Method method, Objec
         activeSpan.log(t);
     }
 
-    private String getRequestURI(String uri) throws MalformedURLException {
+    private String getRequestURI(String uri) {
         if (isUrl(uri)) {
-            String requestPath = new URL(uri).getPath();
+            String requestPath;
+            try {
+                requestPath = new URL(uri).getPath();
+            } catch (MalformedURLException e) {
+                requestPath = uri;
```

Comment:
@xiaqi1210 Let's put in this way. First of all, are you only facing issues when the URL is illegal? Rather than other things. Let's focus on the issue first.

Reply:
ok。So here you think try..catch is fine, just that in the catch block it should be changed to set `requestPath="_blank"`, right?

Reply:
Yes, I am.

Reply:
ok, I will push a new version of code later.